src/components/IndexHeader/Nav/index.js

```diff
@@ -82,4 +68,8 @@ class IndexHeaderNav extends Component {
   }
 }
 
-export default IndexHeaderNav;
+export default connect(
```

Comment:
Just a minor note on consistency, in some components we use the decorator style, and in others we don't.  I think we should stick to one style.  IMHO, I prefer to not use decorators.  😃